### ISSUE TITLE
Online public folder not in sync with Windows 7 pubic folder

### ISSUE THREAD

**Issue by viamyeye** [keybase/client#4943]
Example: 

https://keybase.pub/jubes/ - This shows a 'Test' directory which I initially setup a few weeks back when I first installed keybase on one of my machines.

K:\public\jubes (Windows) and /keybase/public/jubes (Mint 18) - These are current and in sync

**Comment by strib** [comment]
It seems that you've reset your account since making that folder, and keybase.pub is still caching the old pre-reset version.  This is a known bug, and we currently have to restart KBFS on keybase.pub to fix it.  Unfortunately I can't do that from home, so it might have to wait until Monday.  Sorry for the inconvenience!

**Comment by strib** [comment]
> I should just sit back, relax and watch as the keybase glory unfolds?

Yep!  Please wear sunglasses though and let us know if you're pregnant or have any known heart conditions.

**Comment by strib** [comment]
@viamyeye I just restarted keybase.pub, sorry for the delay.  Does it look ok now?

**Comment by viamyeye** [comment]
Hi @strib, apologise for the delay in response. Your actions resolved my issue, has the root cause bug now been resolved?

**Comment by strib** [comment]
The root cause is being worked on right now actually, but is not live yet.
